Python/xPotsSymbol.py

```diff
@@ -91,13 +91,13 @@ def OnFirstUpdate(self):
         global listSDL
         
         AgeStartedIn = PtGetAgeName()
-        if not (type(sdlSaveCloth1.value) == type("") and sdlSaveCloth1.value != "")\
-        or not (type(sdlSaveCloth2.value) == type("") and sdlSaveCloth2.value != "")\
-        or not (type(sdlSaveCloth3.value) == type("") and sdlSaveCloth3.value != "")\
-        or not (type(sdlSaveCloth4.value) == type("") and sdlSaveCloth4.value != "")\
-        or not (type(sdlSaveCloth5.value) == type("") and sdlSaveCloth5.value != "")\
-        or not (type(sdlSaveCloth6.value) == type("") and sdlSaveCloth6.value != "")\
-        or not (type(sdlSaveCloth7.value) == type("") and sdlSaveCloth7.value != ""):
+        if not sdlSaveCloth1.value\
+        or not sdlSaveCloth2.value\
+        or not sdlSaveCloth3.value\
+        or not sdlSaveCloth4.value\
+        or not sdlSaveCloth5.value\
+        or not sdlSaveCloth6.value\
+        or not sdlSaveCloth7.value:
```

Comment:
`if not all((sdlSaveCloth1.value, sdlSaveCloth2.value, [...])):`
with some better formatting?